apps/omg_eth/test/support/root_chain_helper.ex

```diff
@@ -333,4 +333,18 @@ defmodule Support.RootChainHelper do
 
     deposit_blknum
   end
+
+  def exit_id_from_receipt(%{"logs" => logs}) do
```

Comment:
this could easily live inside the tests where its used

Reply:
Updated in 84d6fc67de63477e104fcce55a7d83c316523957